apps/hash-ai-worker-ts/src/workflows/run-flow-workflow.ts

```diff
@@ -172,6 +172,18 @@ export const runFlowWorkflow = async (
     };
   }
 
+  // Ensure the user has permission to create entities in specified web
+  const userHasPermissionToRunFlowInWeb =
+    await flowActivities.userHasPermissionToRunFlowInWebActivity();
+
+  if (userHasPermissionToRunFlowInWeb.status !== "ok") {
+    return {
+      code: StatusCode.PermissionDenied,
+      message: `User does not have permission to run flow in web ${webId}, because they are permissions: ${userHasPermissionToRunFlowInWeb.missingPermissions.join(`,`)}`,
```

Comment:
```suggestion
      message: `User does not have permission to run flow in web ${webId}, because they are missing permissions: ${userHasPermissionToRunFlowInWeb.missingPermissions.join(`,`)}`,
```